index.html

```diff
@@ -0,0 +1,141 @@
+ <div class="languages-2">
+          <button type="button" class="mark-up-2">HTML/CSS</button>
+          <button type="button" class="react-2">Ruby on Rails</button>
+          <button type="button" class="responsive-2">JavaScript</button>
+        </div>
+        </div>
+          <button type="button" class="show-works-2">
+          See this work here &#x2192
+          </button>
+        </div>
+      </div>
+
+    <div class="project-3">
+      <img src="images/project 3.png" alt="" class="project3">
+      <div class="work-3">
+ <h2 class="third-project">Project name goes here</h2>
+<div class="languages-3">
+          <button type="button" class="mark-up-3">HTML/CSS</button>
+          <button type="button" class="react-3">Ruby on Rails</button>
+          <button type="button" class="responsive-3">JavaScript</button>
+        </div>
+      </div>
+      <button type="button" class="show-works-3">
+          See this work here &#x2192
+          </button>
+         </div>
+
+          <div class="project-4">
+      <img src="images/Project 4.png" alt="" class="project4">
+      <div class="work-4">
+ <h2 class="forth-project">Project name goes here</h2>
+<div class="languages-4">
+          <button type="button" class="mark-up-4">HTML/CSS</button>
+          <button type="button" class="react-4">Ruby on Rails</button>
+          <button type="button" class="responsive-4">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-4">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>
+
+<div class="project-5">
+      <img src="images/Project 5.png" alt="" class="project5">
+      <div class="work-5">
+ <h2 class="fifth-project">Project name goes here</h2>
+<div class="languages-5">
+          <button type="button" class="mark-up-5">HTML/CSS</button>
+          <button type="button" class="react-5">Ruby on Rails</button>
+          <button type="button" class="responsive-5">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-5">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>
+
+    <div class="project-6">
+      <img src="images/Project 6.png" alt="" class="project6">
+      <div class="work-6">
+ <h2 class="sixth-project">Project name goes here</h2>
+<div class="languages-6">
+          <button type="button" class="mark-up-6">HTML/CSS</button>
+          <button type="button" class="react-6">Ruby on Rails</button>
+          <button type="button" class="responsive-6">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-6">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>     
+    </section>
+
+    <section class="about-me">
+      <h1 class="me">About me</h1>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
```

Comment:
Please update your about header to be the same as the one in Figma ("About). Also, update your line height to be the same as the one displayed on the original design.

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198788152-2f91a6c4-38ed-4dfb-8293-2725b8e1a88c.png)  | ![image](https://user-images.githubusercontent.com/42799579/198787390-0317c325-ba55-4f50-9e03-8a6c72d8a8f3.png)  |

Here is the typography for you 😉:
![image](https://user-images.githubusercontent.com/42799579/198788360-63493594-9df6-4cc8-895d-bca4797340c4.png)

Reply:
Hi @Gopxfs. Thanks for your feedback. I tried to use the fonts of the figma templates but they didn't pass the linters check. How can I go about it?

Reply:
> Hi @Gopxfs. Thanks for your feedback. I tried to use the fonts of the Figma templates but they didn't pass the linters check. How can I go about it?

Hello @elarhadu. To use custom fonts you need to add them to your project 😄. The easiest way to do it is by adding the font to your HTML `head`. I will explain to you a way of doing this using [google fonts](https://fonts.google.com/):

- Search for the font that you want:
![image](https://user-images.githubusercontent.com/42799579/198803592-001b8e4a-8f09-40bb-b094-d8196a4bcdab.png)

- Select it:
![image](https://user-images.githubusercontent.com/42799579/198803724-bb8ef580-55e5-454c-a351-25f326eb2992.png)

- You will be taken to the font page, scroll down until you have the weights:
![image](https://user-images.githubusercontent.com/42799579/198804057-63e483f4-fc4e-469b-b9ac-cf6d611be885.png)

- Select the ones that you want by clicking here:
![image](https://user-images.githubusercontent.com/42799579/198805635-c8e31d9e-82f6-4c72-8bdf-ffe170f5683f.png)

- A box will appear on the right, after selecting all the fonts copy the link:
![image](https://user-images.githubusercontent.com/42799579/198804518-6865e9ea-f5da-4636-9778-a7b3f6bc12a2.png)

And add it to your HTML `head` 😉.